demo/lexer_demo.cpp

```diff
@@ -1,7 +1,110 @@
 #include <iostream>
+#include "fmt/core.h"
+#include "fmt/color.h"
+#include "strings.hpp"
 #include "lexer.hpp"
+#include "token.hpp"
+#include "strings.hpp"
+
+std::string to_string(lexer::TokenType type) {
+    std::vector<std::string> list{
```

Comment:
Doesn't this allocate memory for the entire vector every time the function is called? Why not put outside the function or make it `static`?

Reply:
Cuz it's a quick-hack function I didn't really bother to think about :)